templates/confirmation/confirm_new_email_address_body.txt

```diff
@@ -0,0 +1,16 @@
+Hi there,
+
+Can you please confirm the ownership of this email address by clicking the
+link below?
```

Comment:
subject: [Zulip] Confirm your new email address for {{ realm }}

Hi!

Our records show that you requested a change to the email address that you use to sign in to Zulip. Please click the link below to confirm that this is the right address.
{{ activate_url }}

{% if verbose_support_offers %}
Feel free to give us a shout at <{{ support_email }}> if you have any questions.
{% else %}

If you did not request this change, please contact your Zulip administrator at <{{ support_email }}>.
{% endif %}

Cheers,

The Zulip Team